scraper/Application Process - 1.yaml

```diff
@@ -1,7 +1,5 @@
 categories:
 - Application Process
-- Scholarships and Financial Aid
```

Comment:
How are you planning to train the ChattterBot read several yaml files ? Further investigation might be needed.

Reply:
I saw your main.py and got my answer. I wonder if training it sequentially  on several different .yaml files makes any difference?

Reply:
I do make a difference (in a bad way). I still don't know why maybe I need to read more on how the training process works.

Reply:
Initial training is successfully completed, but the bot is not robust. Small changes in the sentence results in very suboptimal outputs. Will open an issue to do some analysis.

Reply:
Yes I believe after scraping the data we should also manually include variations of the question (manpower needed 😂)